pkg/controller/handlers/threads/threads.go

```diff
@@ -579,3 +579,28 @@ func (t *Handler) RemoveOldFinalizers(req router.Request, _ router.Response) err
 	}
 	return nil
 }
+
+func (t *Handler) MigrateProjectModelSettings(req router.Request, _ router.Response) error {
```

Comment:
TODO: add a comment to this

Reply:
We don't need to do this. It's only on main and doesn't have a UI (until your changes go in), so no one is actually using the old fields. Just drop them.

Reply:
this is fantastic news